src/bookmark.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/styles.css" />
+    <title>Bookmarks</title>
+  </head>
+  <body class="app-body">
+    <header class="app-header">
+      <h1 class="app-header__heading">Quiz App</h1>
```

Comment:
Maybe change the header with `Bookmark` heading?
```suggestion
      <h1 class="app-header__heading">Bookmark</h1>
```